csharp/extractor/Semmle.Extraction.CSharp.Standalone/BuildAnalysis.cs

```diff
@@ -100,28 +71,41 @@ public BuildAnalysis(Options options, IProgressMonitor progress)
                 dllDirNames.Add(runtimeLocation);
             }
 
-            // TODO: remove the below when the required SDK is installed
-            using (new FileRenamer(sourceDir.GetFiles("global.json", SearchOption.AllDirectories)))
+            if (options.UseMscorlib)
             {
-                var solutions = options.SolutionFile is not null ?
-                        new[] { options.SolutionFile } :
-                        sourceDir.GetFiles("*.sln", SearchOption.AllDirectories).Select(d => d.FullName);
+                UseReference(typeof(object).Assembly.Location);
+            }
+
+            packageDirectory = new TemporaryDirectory(ComputeTempDirectory(sourceDir.FullName));
 
-                if (options.UseNuGet)
+            if (options.UseNuGet)
```

Comment:
Was there a specific reason why this was moved?

Reply:
There were multiple `if (options.UseNuGet)` in this method. I grouped them together.